.github/workflows/pull-request-main-ci.yml

```
@@ -0,0 +1,24 @@
+name: Workflow - Pull Request Main
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+jobs:
+  lint-cargo:
+    if: github.event.pull_request.draft == false
+    uses: ./task-lint-cargo.yaml
```

actionlint (1.7.4)

could not read reusable workflow file for "./task-lint-cargo.yaml": open /home/jailuser/git/task-lint-cargo.yaml: no such file or directory

(workflow-call)